src/mame/shared/xbox_usb.cpp

```diff
@@ -89,12 +89,10 @@ void ohci_usb_controller::write(offs_t offset, uint32_t data)
 {
 	uint32_t old = ohcist.hc_regs[offset];
 
-#ifdef LOG_OHCI
 	if (offset >= 0x54 / 4)
-		logerror("usb controller 0 register HcRhPortStatus[%d] write %08X\n", (offset - 0x54 / 4) + 1, data);
+		m_maincpu->machine().logerror("usb controller 0 register HcRhPortStatus[%d] write %08X\n", (offset - 0x54 / 4) + 1, data);
 	else
-		logerror("usb controller 0 register %s write %08X\n", usbregnames[offset], data);
-#endif
+		m_maincpu->machine().logerror("usb controller 0 register %s write %08X\n", usbregnames[offset], data);
```

Comment:
Same here – the logging is no longer conditional.

Reply:
Ibid.